tpm2/tpm2.go

```diff
@@ -69,10 +69,13 @@ type Command[R any, PR *R] interface {
 }
 
 // PolicyCommand is a TPM command that can be part of a TPM policy.
-type PolicyCommand interface {
+type PolicyCommand[R any, PR *R] interface {
```

Comment:
Couple options come to mind here:

1. We can toss the (non-error) response from the policy command, assuming that users who want to consume the output of the Policy command are not doing so using this interface.
2. We could make this interface function just return any, error and leave it to the caller to do the type assertion if they wish to consume the data.

What do you think? Do either of these options break your use-case?